custom_components/huesyncbox/__init__.py

```diff
@@ -47,7 +47,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except aiohuesyncbox.Unauthorized as err:
         raise ConfigEntryAuthFailed(err) from err
     except aiohuesyncbox.RequestError as err:
-        raise ConfigEntryError(err) from err
+        raise ConfigEntryNotReady(err) from err
     finally:
         if not initialized:
             await api.close()
```

Comment:
The change is tested by checking for the correct state of the config entry. The exception can not be caught directly in a unittest since HA is running the integration.